addons/website_forum/views/website_forum.xml

```diff
@@ -603,7 +617,7 @@
             </span>
             <a t-else="" class="nav-link rounded-pill text-reset" t-attf-href="/forum/#{slug(forum)}/flagged_queue">
                 <i class="fa fa-flag fa-fw"/> Flagged
-                <span t-attf-class="badge pull-right #{forum.count_flagged_posts > 0 and 'badge-danger' or 'badge-light'}" t-esc="forum.count_flagged_posts"/>
+                <span id="count_flagged_posts" t-attf-class="badge pull-right #{forum.count_flagged_posts > 0 and 'badge-danger' or 'badge-light'}" t-esc="forum.count_flagged_posts"/>
```

Comment:
:+1: this was actually broken by https://github.com/odoo/odoo/commit/c8c8eb3d5652e9834d0ecd5aad907ee359edb657 which removed that id by mistake -> the js code couldnt find the element anymore and left some dead JS code

Reply:
You can remove this from your PR, I backported it in the version it was broken...
https://github.com/odoo/odoo/pull/74796